source/frontend/src/features/mapSideBar/management/tabs/activities/ActivitiesTab.tsx

```diff
@@ -43,6 +46,10 @@ export const ActivitiesTab: React.FunctionComponent<IActivitiesTabProps> = ({ ma
           <strong>Activity documentation:</strong> You can attach a document after creating the
           activity. Create, then edit and attach a file if needed.
         </p>
+        <ManagementFileActivitiesListContainer
+          View={ManagementActivitiesListView}
+          managementFileId={managementFile.id}
+        ></ManagementFileActivitiesListContainer>
```

Comment:
Please check with @eddherrera because I thought he has the story about displaying a list of activities?
https://moti-imb.atlassian.net/browse/PSP-8837

It would make sense to include in this PR but please make sure he is not working on the same thing